File: ProgrammersGuide/AsposeCells/WorkingWithWorksheets/ManagementFeatures/ManagingWorksheets/ManagingWorksheetsExamples.cs
using Helpers;
using NUnit.Framework;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using Assert = NUnit.Framework.Assert;
using Description = Microsoft.VisualStudio.TestTools.UnitTesting.DescriptionAttribute;

namespace Examples.ProgrammersGuide.AsposeCells.WorkingWithWorksheets.ManagementFeatures.ManagingWorksheets
{
    [TestClass, TestFixture]
    public class ManagingWorksheetsExamples
    {	
        [TestMethod, Test, Owner("Console")]
        public void AddingWorksheetsToNewExcelFile()
        {
            TestHelper.SetDataDir("ProgrammersGuide/AsposeCells/WorkingWithWorksheets/ManagementFeatures/ManagingWorksheets/AddingWorksheetsToNewExcelFile");

            AddingWorksheetsToNewExcelFileExample.Program.Main();
        }

        [TestMethod, Test, Owner("Console")]
        public void AddWorksheetsToExistingExcelFile()
        {
            TestHelper.SetDataDir("ProgrammersGuide/AsposeCells/WorkingWithWorksheets/ManagementFeatures/ManagingWorksheets/AddWorksheetsToExistingExcelFile");

            AddWorksheetsToExistingExcelFileExample.Program.Main();
        }

        [TestMethod, Test, Owner("Console")]
        public void RemovingWorksheetsUsingSheetName()
        {
            TestHelper.SetDataDir("ProgrammersGuide/AsposeCells/WorkingWithWorksheets/ManagementFeatures/ManagingWorksheets/RemovingWorksheetsUsingSheetName");

            RemovingWorksheetsUsingSheetNameExample.Program.Main();
        }

        [TestMethod, Test, Owner("Console")]
        public void RemovingWorksheetsUsingSheetIndex()
        {
            TestHelper.SetDataDir("ProgrammersGuide/AsposeCells/WorkingWithWorksheets/ManagementFeatures/ManagingWorksheets/RemovingWorksheetsUsingSheetIndex");

            RemovingWorksheetsUsingSheetIndexExample.Program.Main();
        }

    }
}

File: Examples/ProgrammersGuide/AsposeCells/WorkingWithWorksheets/ManagementFeatures/ManagingWorksheets/ManagingWorksheetsExamples.vb
Imports Helpers
Imports NUnit.Framework
Imports Microsoft.VisualStudio.TestTools.UnitTesting
Imports Assert = NUnit.Framework.Assert
Imports Description = Microsoft.VisualStudio.TestTools.UnitTesting.DescriptionAttribute

Namespace Examples.ProgrammersGuide.AsposeCells.WorkingWithWorksheets.ManagementFeatures.ManagingWorksheets
	<TestClass, TestFixture> _
	Public Class ManagingWorksheetsExamples
        <TestMethod(), Test(), Owner("Console")> _
	        Public Sub AddingWorksheetsToNewExcelFile()
	            TestHelper.SetDataDir("ProgrammersGuide/AsposeCells/WorkingWithWorksheets/ManagementFeatures/ManagingWorksheets/AddingWorksheetsToNewExcelFile")
	
	            AddingWorksheetsToNewExcelFileExample.Program.Main()
        End Sub

        <TestMethod(), Test(), Owner("Console")> _
	        Public Sub AddWorksheetsToExistingExcelFile()
	            TestHelper.SetDataDir("ProgrammersGuide/AsposeCells/WorkingWithWorksheets/ManagementFeatures/ManagingWorksheets/AddWorksheetsToExistingExcelFile")
	
	            AddWorksheetsToExistingExcelFileExample.Program.Main()
        End Sub

        <TestMethod(), Test(), Owner("Console")> _
	        Public Sub RemovingWorksheetsUsingSheetName()
	            TestHelper.SetDataDir("ProgrammersGuide/AsposeCells/WorkingWithWorksheets/ManagementFeatures/ManagingWorksheets/RemovingWorksheetsUsingSheetName")
	
	            RemovingWorksheetsUsingSheetNameExample.Program.Main()
        End Sub

        <TestMethod(), Test(), Owner("Console")> _
	        Public Sub RemovingWorksheetsUsingSheetIndex()
	            TestHelper.SetDataDir("ProgrammersGuide/AsposeCells/WorkingWithWorksheets/ManagementFeatures/ManagingWorksheets/RemovingWorksheetsUsingSheetIndex")
	
	            RemovingWorksheetsUsingSheetIndexExample.Program.Main()
        End Sub

	End Class
End Namespace